pkg/trivyk8s/trivyk8s.go

```diff
@@ -60,7 +62,9 @@ func (c *client) ListArtifacts(ctx context.Context) ([]*artifacts.Artifact, erro
 
 		resources, err := dclient.List(ctx, v1.ListOptions{})
 		if err != nil {
-			return nil, fmt.Errorf("failed listing resources for gvr: %v - %w", gvr, err)
+			c.logger.Errorf("failed listing resources for gvr: %v - %w", gvr, err)
+			// if a resource is not found, we log and continue
+			continue
```

Comment:
import "k8s.io/apimachinery/pkg/api/errors"
```suggestion
              if errors.IsNotFound(err) {
			c.logger.Errorf("failed listing resources for gvr: %v - %w", gvr, err)
			// if a resource is not found, we log and continue
			continue
		}
	return nil, fmt.Errorf("failed listing resources for gvr: %v - %w", gvr, err)
```